views/beers.hbs

```diff
@@ -0,0 +1,2 @@
+
+    {{> beerpartial }}
```

Comment:
This is where you should iterate over the beers. 
Wherever you use a partial you pass on the information you want to display there. Which means you will need to add something after the beerpartial for it to work.